### ISSUE TITLE
Training time

### ISSUE THREAD

**Issue by SxJyJay** [chaytonmin/Voxel-MAE#18]
Hi,
   Thanks for your excellent work! I want to know the detailed training configurations (e.g., batch size, GPU num and training time) of your pretraining stage. I have pre-trained on the nuScenes dataset for 9 hours with 8 RTX3090 GPUs of batch size 64, but haven't finished the very first epoch. And I am wondering whether it is a normal phenomenon.

**Comment by chaytonmin** [comment]
> Hi, Thanks for your excellent work! I want to know the detailed training configurations (e.g., batch size, GPU num and training time) of your pretraining stage. I have pre-trained on the nuScenes dataset for 9 hours with 8 RTX3090 GPUs of batch size 64, but haven't finished the very first epoch. And I am wondering whether it is a normal phenomenon.

It is a normal phenomenon as nuScenes dataset is very large.

**Comment by FrontierBreaker** [comment]
Could you please provide the exact batch size, GPU num and training time on nuScenes dataset? Thanks @chaytonmin @IrohXu

**Comment by FrontierBreaker** [comment]
> Hi, Thanks for your excellent work! I want to know the detailed training configurations (e.g., batch size, GPU num and training time) of your pretraining stage. I have pre-trained on the nuScenes dataset for 9 hours with 8 RTX3090 GPUs of batch size 64, but haven't finished the very first epoch. And I am wondering whether it is a normal phenomenon.

Hi, have you succeeded in reproducing the result?

**Comment by SxJyJay** [comment]
> > Hi, Thanks for your excellent work! I want to know the detailed training configurations (e.g., batch size, GPU num and training time) of your pretraining stage. I have pre-trained on the nuScenes dataset for 9 hours with 8 RTX3090 GPUs of batch size 64, but haven't finished the very first epoch. And I am wondering whether it is a normal phenomenon.
> 
> Hi, have you succeeded in reproducing the result?

No, it is too time-consuming to pre-train it with my available GPUs. I think pre-train with 8*80G A100 GPUs is a more rational choice for me in the future.
